### PR TITLE
Update deu.ini

### PR DESCRIPTION
**Pull request by supercrafter333**
Add translation for merge command

### CODE REVIEW

**Comment by HighTecDev** [comment]
This german Translation is ok. Confirm from my Side

File: resources/deu.ini [hunk 1]
```diff
@@ -309,3 +309,16 @@ buy.notforsale=Dieses Grundstück wird nicht verkauft
...
+merge.name=merge
+merge.desc=Füge ein Grundstück in der angegebenen Richtung zusammen
+merge.usage=/p merge [nord|süd|ost|west]
+merge.confirmarg=Bist du dir sicher das du das Plot {%0} {%1} zusammenfügen möchtest? Die änderung ist für immer. Wenn ja, verwende /p merge {%2}
+merge.confirmface=Bist du dir sicher das du das Plot {%0} zusammenfügen möchtest? Die änderung ist für immer. Wenn ja, verwende /p merge confirm
+merge.north=nord
+merge.east=ost
+merge.south=süd
+merge.west=west
+merge.direction=Falsche richtung ausgewählt!
```
Comment (Wertzui123) [inline resources/deu.ini:323]: I'd change this to "Ungültige Richtung angegeben!"

File: resources/deu.ini [hunk 2]
```diff
@@ -309,3 +309,16 @@ buy.notforsale=Dieses Grundstück wird nicht verkauft
...
+merge.success=Grundstück {%0} wurde Erfolgreich mit Grundstück {%1} zusammenfügt.
```
Comment (Wertzui123) [inline resources/deu.ini:324]: - "Erfolgreich" has to be "erfolgreich"
- "zusammenfügt" => "zusammengefügt"

File: resources/deu.ini [hunk 3]
```diff
@@ -309,3 +309,16 @@ buy.notforsale=Dieses Grundstück wird nicht verkauft
 buy.confirm=Möchtest du wirklich das Grundstück {%0} für ${%1} kaufen? Um es zu bestätigen, benutze /p buy confirm
 buy.success=Du hast das Grundstück {%0} erfolgreich für ${%1} erworben
 buy.sold=Der Spieler {%0} hat dein Grundstück ({%1}) für ${%2} gekauft
+
+; The /p merge Subcommand
+merge.name=merge
+merge.desc=Füge ein Grundstück in der angegebenen Richtung zusammen
+merge.usage=/p merge [nord|süd|ost|west]
+merge.confirmarg=Bist du dir sicher das du das Plot {%0} {%1} zusammenfügen möchtest? Die änderung ist für immer. Wenn ja, verwende /p merge {%2}
```
Comment (Wertzui123) [inline resources/deu.ini:317]: - and we're using "Grundstück" instead of "Plot"
- "änderung" has to "Änderung"
- "sicher das" has to be "sicher, dass"

**Review by Wertzui123 — COMMENTED** [review]
All in all good translations with some minor mistakes.

**Comment by supercrafter333** [comment]
The translation is confirmed.

**Comment by Wertzui123** [comment]
No, you haven't fixed all issues in the language file.

**Comment by Wertzui123** [comment]
Please change "Ungültige richtung ausgewählt!" to "Ungültige Richtung angegeben".
And you are still using "Plot" instead of "Grundstück".

**Comment by supercrafter333** [comment]
Now it's okay 😁

**Comment by Wertzui123** [comment]
Yes, I can confirm this translation to be good.

**Pull request merged**